olp-cpp-sdk-dataservice-read/src/generated/api/StreamApi.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <string>
+
+#include <boost/optional.hpp>
+#include <olp/core/client/ApiError.h>
+#include <olp/core/client/ApiResponse.h>
+#include <olp/dataservice/read/ConsumerProperties.h>
+#include "generated/model/SubscribeResponse.h"
+
+namespace olp {
+namespace client {
+class OlpClient;
+class CancellationContext;
+}  // namespace client
+
+namespace dataservice {
+namespace read {
+
+/// Response template type
+template <typename ResultType>
+using Response = client::ApiResponse<ResultType, client::ApiError>;
+
+/**
+ * @brief The `stream` service provides the ability to consume data from a
```

Comment:
```suggestion
 * @brief The `stream` service provides the ability to subscribe to a stream layer and consume data from subscribed layer.
```
Refer to my comment about next sentence